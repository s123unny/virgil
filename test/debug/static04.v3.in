b ../core/static04.v3 6
b ../core/static04.v3 7
b ../core/static04.v3 8
b ../core/static04.v3 9
b ../core/static04.v3 12
b ../core/static04.v3 13
b ../core/static04.v3 14
b ../core/static04.v3 17
run
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
q
