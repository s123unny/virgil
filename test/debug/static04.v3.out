breakpoint|../core/static04.v3 6|0
breakpoint|../core/static04.v3 7|1
breakpoint|../core/static04.v3 8|2
breakpoint|../core/static04.v3 9|3
breakpoint|../core/static04.v3 12|4
breakpoint|../core/static04.v3 13|5
breakpoint|../core/static04.v3 14|6
breakpoint|../core/static04.v3 17|7
stopOnBreakpoint
bt|static04.main|../core/static04.v3|6
variable|1|arg|0|int|false
stopOnBreakpoint
bt|static04.main|../core/static04.v3|7
variable|1|arg|0|int|false
stopOnBreakpoint
bt|static04.main|../core/static04.v3|8
variable|1|arg|0|int|false
stopOnBreakpoint
bt|static04.main|../core/static04.v3|9
variable|1|arg|0|int|false
end
result|0
